packages/ember-htmlbars/lib/system/build-component-template.js

```diff
@@ -186,7 +186,7 @@ function normalizeComponentAttributes(component, attrs) {
     }
   }
 
-  normalized.role = buildStatement('get', 'ariaRole');
+  normalized.role = normalized.role || buildStatement('get', 'ariaRole');
```

Comment:
However it seems a bit confusing that there is two ways of achieving the same thing. `ariaRole="foo"` or `role="foo"` + attributeBindings.
I think that it should be only one blessed approach and the other one should be deprecated.